pgjdbc/src/main/java/org/postgresql/jdbc/TimestampUtils.java

```diff
@@ -830,9 +830,8 @@ public synchronized String toString(LocalDateTime localDateTime) {
       return "-infinity";
     }
 
-    // LocalDateTime is always passed with time zone so backend can decide between timestamp and timestamptz
-    ZonedDateTime zonedDateTime = localDateTime.atZone(getDefaultTz().toZoneId());
-    return toString(zonedDateTime.toOffsetDateTime());
+    OffsetDateTime offsetDateTime = localDateTime.atOffset(ZoneOffset.UTC);
```

Comment:
I think we should do the opposite there. `LocalDateTime` has no time zone, we should not make up one. I would simply not send a time zone if we don't have one.